crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -35,7 +36,27 @@ impl DynamicScene {
         let mut scene = DynamicScene::default();
         let type_registry = type_registry.read();
 
+        // Resources archetype
+        let resources_archetype = world.archetypes().resource();
+        for component_id in resources_archetype.components() {
+            let reflect_resource = world
+                .components()
+                .get_info(component_id)
+                .and_then(|info| type_registry.get(info.type_id().unwrap()))
```

Comment:
Maybe skip components without a type id instead of panicking

Reply:
Done in 0fd72ae